joft/utils.py

```diff
@@ -26,7 +42,23 @@ def load_toml_app_config() -> typing.Any:
     for path in possible_paths:
         config_file_path = pathlib.Path(path) / "joft.config.toml"
         if config_file_path.is_file():
-            break
+            if config := read_and_validate_config(config_file_path):
```

Comment:
This can raise and that is never handled.

Suggestion: Replace if-else with try-except and display the exception type+message in the user-facing error.